### ISSUE TITLE
Every X Days daily displaying on the wrong day.

### ISSUE THREAD

**Issue by deathau** [HabitRPG/habitrpg-android#385]
I have a daily set up for every 14 days from 06/17/2015 (a Wednesday). In the app, it shows up every second Tuesday, the day before it's supposed to show up. Web is fine, and the old app was fine.
My current theory is a timezone issue (I'm in GMT+10, +11 at the moment due to daylight savings).
Galaxy note 3, Android 5.0.

If you need any more information, let me know.

**Comment by vIiRuS** [comment]
Can you check in the android app what it displays as the start date? My guess is that it interprets the date wrong (yay timezones) and thus uses the 16.6.2015 as the start date.

**Comment by deathau** [comment]
Indeed, you are correct. According to the app, it started on the 16th.
Yay timezones indeed

**Comment by vIiRuS** [comment]
As timezones are evil magic, I'm not 100% sure it's really gone, but at least in my cases the problems were resolved. If this still happens after the next update, let me know.

**Comment by deathau** [comment]
The problem has been resolved in the latest update.
Thanks vIiRuS

**Comment by amiyuy** [comment]
I have an every 2 days daily.

Web shows fine, mobile shows on the wrong day. It showed as grey today on web and due on mobile.

Start date shows as 12/24/2015 on both web and mobile.

I switched the start date to 12/25/2015 and it became due on web and grey on mobile.  Both web and mobile showed the start date as 12/25/2015.

I am in the Pacific Time Zone (UTC−08:00-current- and UTC−07:00).  My daily start time is 5AM.

Previously if I refreshed I'm pretty sure it eventually updated so I assumed it was just a refresh error, but it's not doing that right now and that I can change the date and it switches between due/grey makes me think it's related to this.
